packages/waku/src/config.ts

```diff
@@ -58,6 +58,7 @@ export interface Config {
    */
   unstable_honoEnhancer?: string | undefined;
   /**
+   * @deprecated use `vite` instead.
```

Comment:
I wonder if we can deprecate it in advance, but maybe it requires Environment API?

Reply:
For current multi server/builder setup, single `vite` config is not flexible enough and I think `unstable_viteConfigs` approach is more appropriate.

Reply:
Makes sense.